docs/submit-transactions.md

```diff
@@ -16,13 +16,13 @@ and behaviour of each one of them.
 
 ⓘ All the transactions provide the same [Command-Line Options](#command-line-options).
 
-- Transfer funds from the author's account to another account, which can be a local, org's, or user's account.
+- Transfer funds from the author's account to another account, which can be a user, an org or a local key-pair.
```

Comment:
This sentence contains the statement “an account can be a user” and “an account can be a local key pair”. It also sounds like a user, an org an a local key pair a kind of similar. This might be confusing. In fact an account may be associated to a user or an org but it need not be. And it also does not need to be a local key pair

Reply:
yeah, I agree with you. Before it was clear it was the org/user/pair**'s** account but this change breaks it. We could simplify and just have this?

```
Transfer funds from the author's account to another account
```